client/http/client.go

```diff
@@ -695,10 +723,39 @@ func (c *client) AccelerateScheduleInBatch(ctx context.Context, keyRanges []*Key
 	}
 	return c.requestWithRetry(ctx,
 		"AccelerateScheduleInBatch", AccelerateScheduleInBatch,
-		http.MethodPost, bytes.NewBuffer(inputJSON), nil)
+		http.MethodPost, inputJSON, nil)
+}
+
+// GetSchedulers gets all scheduler names.
+func (c *client) GetSchedulers(ctx context.Context) ([]string, error) {
+	var schedulers []string
+	err := c.requestWithRetry(ctx,
+		"GetSchedulers", Schedulers,
+		http.MethodGet, nil, &schedulers)
+	if err != nil {
+		return nil, err
+	}
+	return schedulers, nil
+}
+
+// PostSchedulerDelay changes the delay of given scheduler.
+func (c *client) PostSchedulerDelay(ctx context.Context, scheduler string, delaySec int64) error {
```

Comment:
Do we need tests for these two interfaces?

Reply:
Sorry currently I don't know how to create the scheduler in http client tests so they can not be covered.

Reply:
It will be available at https://github.com/tikv/pd/pull/7516/files#diff-138f7d23a1171f829bb57a7edcd66e8ac0ceb81f1cb2f28e497284c732aaad22R67.

Reply:
I can wait #7516 merge first and then add a test covering these new functions. @rleungx please help review that PR

Reply:
updated. PTAL